scripts/copy-files.js

```diff
@@ -15,6 +16,16 @@ function copyFile(file) {
   }).then(() => console.log(`Copied ${file} to ${buildPath}`));
 }
 
+function copyTypings(folder) {
```

Comment:
That sounds better :). I want to reduce the overhead for new contributors. It's not obvious what this file is about. The less code branch we have, the better.